shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/FeedingsState.kt

```diff
@@ -11,5 +12,7 @@ data class FeedingsState(
     val hasReviewedFeedings: Boolean = false,
     val isListLoading: Boolean = false,
     val isLockedAndLoading: Boolean = false,
-    val isError: Boolean = false
+    val isError: Boolean = false,
+    val rejectionReasonRequestType: RejectionReasonRequestType? = null,
+    val feedingForRejection: FeedingModel? = null
```

Comment:
This feeding is not used on UI, so it can be moved to private variable in the viewmodel

Reply:
Good point, thank you. Updated.